### ISSUE TITLE
Which torch operator can cause nonzero problems?

### ISSUE THREAD

**Issue by 980202006** [NVIDIA/TensorRT#2201]
There is no torch.nonzero in my torch model, but a nonzero error is reported. Which torch operations may cause nonzero problems, or how can I find out which operators contain nonzero operations?

**Comment by zhiqwang** [comment]
torch.where I guess

**Comment by zerollzeng** [comment]
I think it's better to ask in the pytorch repo.

**Comment by wugoukanle** [comment]
it has 3 scene：
1. tensor mask op，as example：x = x[x.sum(dim=-1) > 0]
2. torch.where(condition), as example: torch.where(x>0)
3. torch.nonzero(x)

**Comment by 980202006** [comment]
Thank you!
